aws-cloudformation/aws-cloudformation-resource-providers-codeartifact
# Boostrapping + Create, List, and Read Handlers for AWS::CodeArtifact::Domain

*Issue #, if available:*

*Description of changes:*

This PR has
+ the initial boostrapping  AWS::Codeartifact::Domain Resource handlers
+ Implementation of Create, Read, List handlers

**Testing**
Unit tests and I ran 
`cfn invoke CREATE sam-tests/create_payload.json` 
`cfn invoke DELETE sam-tests/delete_payload.json`
`cfn invoke LIST sam-tests/create_payload.json`

and submitted the resource and create a domain with a cloudformation stack, testing 
+ Create Stack with/without PolicyText
+ Update Stack with Domain policy
+ Update Stack while removing policy
+ Deleting Stack


Please see the latest commit to see a diff of changes I wrote

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
